index.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+const {Buffer} = require('buffer/');
```

Comment:
`Buffer` is a global. Prefer that one.

Reply:
@sindresorhus , same comment as above (frontend)

Reply:
fixed